src/component/playground/Playground/PlaygroundResultsTable/FeatureResultInfoPopoverCell/PlaygroundResultFeatureStrategyList/PlaygroundResultStrategyList/PlaygroundResultFeatureStrategyItem/PlaygroundResultStrategyExecution/helepers.ts

```diff
@@ -0,0 +1,14 @@
+export const getMappedParam = (key: string) => {
+    switch (key) {
+        case 'userIds':
+        case 'UserIds':
+            return 'userId';
+        case 'hostNames':
+        case 'HostNames':
+            return 'hostname';
```

Comment:
As we discovered, this doesn't actually map in any way, so we should remove this or return undefined or something.